packages/UI/src/components/organisms/Dropdown/Dropdown.tsx

```diff
@@ -47,7 +49,7 @@ const Dropdown = ({
         prev[text] = checked;
         return prev;
       });
-    recieveValue?.(checkedOptions);
+    recieveValue?.(optionsToReturn);
```

Comment:
Just to confirm my understanding, this expression returns `receiveValue` if it's defined. Otherwise it returns `optionsToReturn`?

Reply:
@chungthuang it will always return `optionsToReturn`